<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>701</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="infoUserButton">
       <property name="text">
        <string>Profile info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="choiceBankBox">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeOfCreditBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="timePayoutBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="percentCreditBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="sumCreditBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>82</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="reasonBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="paymentBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
